tcms_settings_dir/trackers_integration.py

```diff
@@ -8,5 +8,6 @@
     not in EXTERNAL_BUG_TRACKERS  # noqa: F821
 ):
     EXTERNAL_BUG_TRACKERS.append(  # noqa: F821
-        "trackers_integration.issuetracker.OpenProject"
+        "trackers_integration.issuetracker.OpenProject",
+        "trackers.integration.issuetracker.Mantis",
```

Comment:
`.append()` takes a singular value.  Repeat the same code block for the Mantis bug tracker b/c we also need to be sure it hasn't been added previously.

Reply:
Also see `issuetracker/__init__.py`

Reply:
Ooops missed that, it is a copy paste error from the PR on Kiwi repo. 

Will fix it!

Is there any way to run the trackers-integration standalone, manage.py can’t find tcms folder and there are no targets in make file?